public/javascripts/SVValidate/src/modal/ModalMissionComplete.js

```diff
@@ -34,7 +35,7 @@ function ModalMissionComplete (uiModalMissionComplete, user, language = 'en') {
         // Have to remove the effect since keyup event did not go through (but no keyboard use on mobile).
         if (svv.keyboard) {
             svv.keyboard.removeAllKeyPressVisualEffect();
-            svv.keyboard.enableKeyboard();
+            //svv.keyboard.enableKeyboard();
```

Comment:
Instead of commenting something out, just delete it! If we don't need it, then we don't need it :)